src/pages/ShowMovie.css

```diff
@@ -0,0 +1,36 @@
+.detailPageContainer{
+  position: relative;
+  width: 100%;
+  height: 100vh;
+}
+
+.summary{
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 50px;
+  width: 100%;
+  max-width: 400px;
+  box-sizing: border-box;
+  position: absolute;
+  bottom: 0;
```

Comment:
This doesn't work too well on mobile:

<img width="603" alt="Skärmavbild 2024-04-10 kl  18 28 12" src="https://github.com/Technigo/project-movies-vite/assets/51529201/66e9f44d-cbb4-41a1-974d-565e7d1a07ee">

Reply:
Fixed now! :)